metashare/api/models.py

```diff
@@ -54,6 +54,7 @@ class User(HashIdMixin, AbstractUser):
     objects = UserManager()
     currently_fetching_repos = models.BooleanField(default=False)
     devhub_username = StringField(null=True, blank=True)
+    use_global_devhub = models.BooleanField(default=False)
```

Comment:
Users should always use the sitewide DEVHUB_USERNAME as long as it is set and the user isn't configured to use a different one (via the user's devhub_username field or an oauth-connected Salesforce account). So this isn't the field we need to add.

The field we discussed adding last Thursday was a field that we as admins can set to allow the user to override the sitewide default by connecting their own devhub. Maybe call it `allow_devhub_override` (boolean, default: False). This would influence whether the user can see the existing UI for connecting a dev hub. The UI should be shown if DEVHUB_USERNAME is not set, or if user.allow_devhub_override is True.